Configurable analog head size

iota97 opened the pull request:
Change the default a bit as now you have to press on the head to get the input (before pressing on the background would move the stick as well).

Also add a checkbox to disable the stick background if someone want to scale it pretty big (it's still being drawn in the layout editor to give feedback).

LunaMoo commented:
> Change the default a bit as now you have to press on the head to get the input (before pressing on the background would move the stick as well).


 Don't think changing the default of something like that is a good idea a lot of people who actually use only touch controls might be used to simply clicking on the side instead of having to click on the center before allowed to drag.

iota97 commented:
You got quite a point.

I guess it's possible to make the head collision a bit larger than the image to match the default. Not sure how much relevant is if the image is off of a 15% or so tho'.

hrydgard commented:
This needs a rebase.

I'd like the default to not change behavior or look at all, and I'm not sure if it makes sense to have separate settings for left and right stick, given how rare second stick usage is.

iota97 committed:
Configurable analog head size

iota97 commented:
Rebased and made the default the same (just to note in master the stick "collision" size ignored the scale so I fixed that).

About having 2 different setting for right and left I would just find confusing if changing an analog would change both in the layout screen. I can make them the same if you prefer tho', I got the bad habits of adding too many setting xD

iota97 committed:
Fix opacity

unknownbrackets commented:
I think having separate settings for left and right makes sense, given that the right analog sizing is within the control management anyway.

-[Unknown]
